smart_card_connector_app/src/chrome-api-jstocxxtest.js

```diff
@@ -801,242 +782,244 @@ goog.exportSymbol('testChromeApiProviderToCpp', {
           /*expectedProtocol=*/ chrome.smartCardProviderPrivate.Protocol.T1);
     },
 
-    'tearDown': async function() {
-      expectReportPlainResult(/*requestId=*/ 68, 'SUCCESS');
-      await mockChromeApi
-          .dispatchEvent(
-              'onDisconnectRequested', /*requestId=*/ 68, readerHandle,
-              chrome.smartCardProviderPrivate.Disposition.LEAVE_CARD)
-          .$waitAndVerify();
+    'tearDown': function() {
       readerHandle = null;
     },
 
-    // Test that Status returns information about the card.
-    'testStatus': async function() {
-      expectReportStatusResult(
-          /*requestId=*/ 111,
-          SimulationConstants.GEMALTO_PC_TWIN_READER_PCSC_NAME0,
-          chrome.smartCardProviderPrivate.ConnectionState.NEGOTIABLE,
-          chrome.smartCardProviderPrivate.Protocol.T1,
-          SimulationConstants.COSMO_ID_70_ATR, 'SUCCESS');
-      await mockChromeApi
-          .dispatchEvent('onStatusRequested', /*requestId=*/ 111, readerHandle)
-          .$waitAndVerify();
-    },
+    'testDisconnectOnTearDown': {
```

Comment:
I was thinking about this too, but there's a problem that it becomes increasingly hard, when looking at a test case, to understand which group it belongs to.

Maybe solve it like this, by moving the test cases to a flat list of constants?

```
const SIMPLE_TEST_CASES = {
  'testSmoke': async function() { <...> },
  'testEstablishContext': async function() { <...> },
};
const DISCONNECT_ON_TEAR_DOWN_TEST_CASES = {
  'testSmoke': <...>
  <...>
};
goog.exportSymbol('testChromeApiProviderToCpp', {
  'setUp': async function() { <...> },
  <...>
  ...SIMPLE_TEST_CASES,
});
```

Reply:
done